platform/ios/src/MLNMapView+Metal.mm

```diff
@@ -82,7 +82,7 @@ void swap() override {
 
         // Un-comment for synchronous, which can help troubleshoot rendering problems,
         // particularly those related to resource tracking and multiple queued buffers.
-        //[commandBuffer waitUntilCompleted];
+        [commandBuffer waitUntilCompleted];
```

Comment:
Probably want to turn this back off.
It might be changing the frame rendering time too.

Reply:
oops. correct. remeasuring.